backend/spec/features/admin/configuration/logo_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe "Admin", type: :feature, js: true do
+  stub_authorization!
+
+  let!(:store) do
+    create(:store, name: 'Test Store', url: 'test.example.org',
+           mail_from_address: 'test@example.org')
+  end
+
+  before(:each) do
+    visit spree.admin_path
+    click_link "Settings"
+  end
+
+  context "visiting the main admin page" do
+    it "sets the ID on the header's logo" do
+      expect(page).to have_selector("img#admin-nav-logo", count: 1)
+    end
+  end
+end
```

Comment:
Having `js:true` over there makes this test very slow, compared to the benefit of testing that an ID is present, I'd think that's a negative gain of having the test. So, I'd rather choose not to have it.